mypy/main.py

```diff
@@ -555,18 +558,12 @@ def create_source_list(files: Sequence[str], options: Options) -> List[BuildSour
     targets = []
     for f in files:
         if f.endswith(PY_EXTENSIONS):
-            try:
-                targets.append(BuildSource(f, crawl_up(f)[1], None))
-            except InvalidPackageName as e:
-                fail(str(e))
+            targets.append(BuildSource(f, crawl_up(f)[1], None))
```

Comment:
I don't think it's necessarily an improvement to let `InvalidPackageName` (raised by `crawl_up()`) is now bubbled out from this. Maybe add a comment here, and another to the function header (in a docstring?) about which exception is raised if things go pear-shaped?